From fd11241003c92f8709f32c66f3ca3b4e8ff8fee5 Mon Sep 17 00:00:00 2001
Message-Id: <fd11241003c92f8709f32c66f3ca3b4e8ff8fee5.1577454401.git.liu.denton@gmail.com>
In-Reply-To: <cover.1577454401.git.liu.denton@gmail.com>
References: <cover.1577454401.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Mon, 9 Sep 2019 13:17:02 -0700
Subject: [PATCH 07/16] t3030: use test_path_is_missing()
To: Git Mailing List <git@vger.kernel.org>

Previously, we would use `test_must_fail test -d` to ensure that the
directory is removed. However, test_must_fail() should only be used for
git commands. Use test_path_is_missing() instead to check that the
directory has been removed.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 t/t3030-merge-recursive.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/t3030-merge-recursive.sh b/t/t3030-merge-recursive.sh
index 2170758e38..d48d211a95 100755
--- a/t/t3030-merge-recursive.sh
+++ b/t/t3030-merge-recursive.sh
@@ -604,7 +604,7 @@ test_expect_success 'merge removes empty directories' '
 	git commit -mremoved-d/e &&
 	git checkout master &&
 	git merge -s recursive rm &&
-	test_must_fail test -d d
+	test_path_is_missing d
 '
 
 test_expect_success 'merge-recursive simple w/submodule' '
-- 
2.24.1.810.g65a2f617f4

